frontend/src/lib/components/icon/icons/AddToListIcon.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import type {IconProps} from '../Icon.type';
+  import {IconBase} from '../base';
```

Comment:
That was only because the current version included the generic version of `Icon` for demonstration purposes, but that won't anyway be a problem with lazy-loading version.